tutorials/sar-basics/index.md

```diff
@@ -0,0 +1,181 @@
+
+A key point about SAR is that it is a radar (RAdio Detection And Ranging) instrument, and it's basic measurements are intensity and phase measurements of the backscattered signal, sampled in time bins along the **azimuth** (across track direction of the sensor antenna) and **range** (perpendicular to the direction of the sensor antenna). These time bins relate to locations on Earth from which the backscattered signal originate. The "Synthetic Aperture" of the SAR is the virtual creation of a huge antenna which makes use of the sensor motion and the associated complex data processing, which is necessary to create high resolution in azimuth. By moving along the orbit, the next azimuth line is created from a new microwave pulse in the range direction (this is oversimplified).
+
+The key difference between SAR and optical sensors is that
+
+- SAR is an **active** sensor, transmitting **microwave** radiation, for which it receives the portion **scattered back** to the sensor. Optical sensors are **passive** sensors that register reflected radiation from the Sun (in specific parts of the visible and infrared spectrum).
+- SAR is **side-looking**, unlike optical sensors which are, usually, nadir looking. 
+- SAR radiation is **coherent**, within a pre-set range of wavelengths. This is useful, because it allows precise **phase** and **intensity** measurements and the use of **polarization**. However, it is also the cause of **speckle**. Optical sensors are not coherent, apart from laser and lidar instruments. 
+
+Reflecting on these differences is important, because it is the basis to understand the relative advantages of SAR compared to optical sensors:
+
+- SAR does not depend on Sun light, thus works **day and night**, provided the sensor acquires data (i.e. is switched on)
+- SAR is **insensitive to the atmosphere** in C-band (Sentinel-1, except for very dense rain cells) and L-band (ALOS-PALSAR), both of which are in GEE collections. 
+- SAR backscattering (intensity) depends on **different physical properties of the "target"** compared to the properties that cause reflectance in optical sensors. These properties relate to the structural geometry and (electromagnetic) material properties of what is illuminated by the incident radiation.
+- SAR data can be **calibrated**, without the need for atmospheric correction, leading to **consistent time series**
+
+as well as the disadvantages:
+
+- The coherent nature of the SAR microwave radiation causes **speckle**. This causes the "salt and pepper" appearance of extended target areas (e.g. a large homogeneous agricultural field) that one would expect to have a constant backscattering behavior. Speckle can be reduced in different ways (see next tutorial), but is difficult to eliminate. 
```

Comment:
salt-and-pepper

(Compound adjectives are supposed to be hyphenated. Tons of English speakers don't do it, but I think this enhances readability.)

Reply:
revised